tailwind.config.js

```diff
@@ -1,8 +1,23 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-    content: ["*.html"],
-    theme: {
-      extend: {},
+  content: ["./src/**/*.{html,js}"],
+  theme: {
+    extend: {
+      colors: {
+        darkPrimary: "#116355",
+        primary: "#8ACC50",
+        secondary: "#F1F4F6",
+        textColorblack: "#000000",
+        textColorwhite: "#FFFFFF",
+        backgroundColor: "#001342",
+        yellow: "#F4BA00",
```

Comment:
Most of the colors exist in tailwind and don't really know why do you need this here.